build.gradle

```diff
@@ -130,6 +130,9 @@ tasks.withType(FindBugs) {
   }
 }
 
+check.dependsOn jacocoTestReport
```

Comment:
I'd still remove it.

The other line on `jacocoTestReport` depending on `test` is ok.